vortex-serde/src/messages.rs

```diff
@@ -48,7 +48,7 @@ impl WriteFlatBuffer for IPCMessage<'_> {
     }
 }
 
-impl<'a> WriteFlatBuffer for IPCSchema<'a> {
```

Comment:
Why doesn't clippy find this? Did we change something?

Reply:
not sure, I just ran `cargo +nightly fix --lib --allow-dirty --allow-staged && cargo +nightly clippy --fix --lib --allow-dirty --allow-staged && cargo +nightly fmt --all` and it elided a number of unnecessary lifetimes